oap-server/server-receiver-plugin/receiver-proto/src/main/fbs/istio/node-info.fbs

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2020 Istio Authors. All Rights Reserved.
```

Comment:
Here we should note the commit SHA of this file located at istio repo. and a tool or shell to update it based on this SHA.

Reply:
> and a tool or shell to update it based on this SHA.

Doing that means we want to download the file on the fly, do we still need the license for this? @wu-sheng we don't include that in source distribution then.

Reply:
@wu-sheng 👆

Reply:
What is the size of downloading. I assume this would make compiling more unstable, especially in China.

Reply:
> What is the size of downloading. I assume this would make compiling more unstable, especially in China.

1.6KB

Reply:
Is it possible to download this single one file?

Reply:
Yes, I'll configure a maven phase to download this before compiling, with a URL format like this, the commit SHA is a property that can be specified (the necessity needs discussion though) `https://raw.githubusercontent.com/istio/proxy/94a7d9942aa8181f688933cb265879d59a0d85dd/extensions/common/node_info.fbs`

Reply:
This path is not accessable from some people, if I remember correctly. I mean githubusercontent.com

Reply:
If that's the case, I'm afraid we have to clone the repo just for that one file. @hanahmily can you elaborate more about why we need an update mechanism to this file? What makes it different from the other files that we copied from, like Envoy, Jaeger, etc.?

Reply:
I am totally fine with the copy and LICENSE change. This file is for sharing, nothing to concern from my understanding. I have many proto files copied from istio/envoy/...

Reply:
I'll just keep it as is, unless @hanahmily give a real use case where we need it

Reply:
Could you leave a comment that implies the location and commit sha? It is too tricky to maintain the codes without them even though the original contributor like you doesn't figure it out after several months.
Syncing is a more complicated topic, as there are a lot of unversioned protocol files in the repo, need to be discussed further.

Reply:
> Could you leave a comment that implies the location and commit sha? It is too tricky to maintain the codes without them even though the original contributor like you doesn't figure it out after several months.
Syncing is a more complicated topic, as there are a lot of unversioned protocol files in the repo, need to be discussed further.

Comment is added